### ISSUE TITLE
Help with this crash and console spam loop

### ISSUE THREAD

**Issue by Trnrr** [haykam821/Lootstrap#1]
Hello I was trying to get this to work and got it to give the item on first join, but right after mc stops responding and I have to close it because its spamming the log file.

Could you check if I have anything wrong here?
Loot tabe in: .minecraft\config\datapacks\Project Nutella\data\lootstrap\loot_tables\first_join.json
first_join.json: https://gist.github.com/Trnrr/4e5dbdd49e6a195aad910f1130efedb3

Error before spam: https://gist.github.com/Trnrr/f90c376aeee9d5975f677549563a0a3d
Another : https://gist.github.com/Trnrr/516cad0ded35ab1a29f15681315b7888
Eventually just spams this: https://gist.github.com/Trnrr/29565880d61c5488ad5ee90100c7426c

**Comment by haykam821** [comment]
A few questions:

* Are there any other datapacks/datapack files other than the Lootstrap one?
* Are you granting yourself lots of recipes at once? I recall the `DivideByZeroException` being thrown if you grant lots of recipes at once.
* Are you on Minecraft 1.15.2, and if not, does this issue still occur on Minecraft 1.15.2?
* Are you using any other mods?

**Comment by Trnrr** [comment]
Ok I am sorry, this issue is probably not relating to your mod because I was just testing and it seems to happen even after I renamed first_join to first_joinn. And one of the times I joined my world a bunch of recipes popped up in the top corner at once so you are probably correct. I just gotta figure out what causes that.

But to answer your questions:
- There are datapacks, Anti_Ender-Grief_1.0.zip, PvP_Player_Heads_1.0.zip, Bat_Membranes_1.0.zip, ShulkerShells_1.0.zip, Enderdragon_elytra_1.0.zip, Tridents_o_Plenty_1.1.zip
- Im not sure maybe a mod is doing it. I will try and find it
- Yes I am on 1.15.2
- Yes many mods and it pains me sometimes: https://gist.github.com/Trnrr/1ef7b84682e8d07f126b51f04d007313

I recently added/updated these so I will look at these first
craftable-horse-armor1.0.0.jar
quazi-modded-1.0.0+1.15.2.jar
lootstrap-1.0.0.jar
mobropes-1.15.1+1.0.1.jar
sheet-lib-1.4.0+1.15.2
 better-wither-skeletons-1.0.0.jar
creeper-spores-1.3.2.jar
LightOverlay-4.2.jar
PresenceFootsteps-r22+1.15.2.jar
terrestria-1.2.0.jar
